vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -60,6 +61,17 @@
     private final DateTimePickerTimePicker timePicker = new DateTimePickerTimePicker();
     private Locale locale;
 
+    private final static SerializableFunction<String, LocalDateTime> PARSER = s -> {
+        return s == null || s.isEmpty() ? null : LocalDateTime.parse(s);
+    };
+
+    private final static SerializableFunction<LocalDateTime, String> FORMATTER = d -> {
+        return d == null ? "" : d.toString();
+    };
```

Comment:
I think the return line here should be;
```java
return d == null ? "" : d.truncatedTo(ChronoUnit.MILLIS).toString();
```

Without this the string result might contain nanoseconds too, which will be ignored as an invalid value on client side when setting to min or max. I ran into this problem when trying my suggested demo improvement above with `LocalTime.MAX` (on client the max was set to empty string) and doing this change to the `FORMATTER` fixed it.

This same approach is used in `TimePicker.setValue()`, but does not seem to be used in the `FORMATTER` of `TimePicker` but I think it should be updated there also.

Reply:
Updated.